apps/form-builder/services/apiService.tsx

```diff
@@ -0,0 +1,293 @@
+import axios from 'axios'
+import {
+  IForm,
+  IFormApplicantType,
+  IFormBuilder,
+  IGroup,
+  IInput,
+  ILanguage,
+  IStep,
+  ItemType,
+} from '../types/interfaces'
+
+const BASEURL = 'https://profun.island.is/umsoknarkerfi/api'
+
+export async function getForm(id: unknown) {
+  try {
+    const response = await axios.get(`${BASEURL}/Forms/${id}`)
+    console.log('getForm response: ', response.data)
+    return response.data
```

Comment:
Should use strong types generated from OpenAPI. E.g. with https://openapi-generator.tech/